controller_interface/src/controller_interface_base.cpp

```diff
@@ -88,6 +95,20 @@ const rclcpp_lifecycle::State & ControllerInterfaceBase::configure()
     update_rate_ = static_cast<unsigned int>(get_node()->get_parameter("update_rate").as_int());
     is_async_ = get_node()->get_parameter("is_async").as_bool();
   }
+  if (is_async_)
+  {
+    const unsigned int thread_priority =
+      static_cast<unsigned int>(get_node()->get_parameter("thread_priority").as_int());
+    RCLCPP_INFO_STREAM(
+      get_node()->get_logger(),
+      "Starting async handler with scheduler priority: " << thread_priority);
```

Comment:
```suggestion
    RCLCPP_INFO(get_node()->get_logger(), "Starting async handler with scheduler priority: %d", thread_priority);
```
better without streaming operators